.github/workflows/gke.yaml

```diff
@@ -1,66 +1,75 @@
 name: ConformanceGKE
 
 on:
-  pull_request_target:
-    types:
-      - "labeled"
-  #pull_request: #uncomment for testing
-  #  types:
-  #    - "labeled"
+  issue_comment:
+    types: [created]
   push:
-    branches:
-      - master
+    branches: [master]
```

Comment:
Just small nit, can we keep it as a "list" instead of `[` and `]`  so it is consistent with the other files?

Reply:
Sure! I liked the compactness, but you're right, consistency comes first.